### PR TITLE
Remove greenkeeper badge, add assistant info

### PR DESCRIPTION
**Pull request by szeck87**
* Removes the greenkeeper badge
* Adds information about the assistant's configuration wizard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ code --install-extension Glavin001.unibeautify-vscode
 
 > Important: You must first configure for the extension to work properly.
 
+Use the Unibeautify assistant, https://assistant.unibeautify.com/, for an interactive setup of your configuration file. Simply select the languages you want, and it will walk you though the options available. At the end simply download or copy to your clipboard your configuration.
```
Comment (Glavin001) [inline README.md]: Same comment as https://github.com/Unibeautify/website/pull/93#discussion_r196138292

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Unibeautify for VSCode
 
-[![Build Status](https://travis-ci.com/Unibeautify/vscode.svg?branch=master)](https://travis-ci.com/Unibeautify/vscode) [![Visual Studio Marketplace](https://img.shields.io/vscode-marketplace/d/Glavin001.unibeautify-vscode.svg)](https://marketplace.visualstudio.com/items?itemName=Glavin001.unibeautify-vscode) [![Greenkeeper badge](https://badges.greenkeeper.io/Unibeautify/vscode.svg)](https://greenkeeper.io/)
+[![Build Status](https://travis-ci.com/Unibeautify/vscode.svg?branch=master)](https://travis-ci.com/Unibeautify/vscode) [![Visual Studio Marketplace](https://img.shields.io/vscode-marketplace/d/Glavin001.unibeautify-vscode.svg)](https://marketplace.visualstudio.com/items?itemName=Glavin001.unibeautify-vscode)
```
Comment (Glavin001) [inline README.md]: Is there a @renovate-bot badge we should add here?

**Comment by szeck87** [comment]
I'm not opposed to that, but I think its too early.  I want to get feedback and make UI/UX changes as necessary, plus the themes/colors may change sooner than later.

**Comment by Glavin001** [comment]
> I think its too early.

Agreed.

**Pull request merged**